<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cargar Opiniones</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1031</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="file_tab">
     <attribute name="title">
      <string>Fichero</string>
     </attribute>
     <widget class="QTableWidget" name="file_tableWidget">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>31</y>
        <width>1001</width>
        <height>381</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Etiqueta</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ruta</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="file_pushButton_add">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>420</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="file_pushButton_load">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>560</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="file_pushButton_clear">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>560</y>
        <width>114</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Borrar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="file_pushButton_selectfolder">
      <property name="geometry">
       <rect>
        <x>783</x>
        <y>520</y>
        <width>231</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleccionar carpeta</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="file_textEdit_path">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>991</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_labels_file">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>420</y>
        <width>191</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Seleccionar etiqueta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Unlabeled</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>etiqueta1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>etiqueta2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>mimammamemimma</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_back">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>560</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Atr√°s</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="URL_tab">
     <attribute name="title">
      <string>URL</string>
     </attribute>
     <widget class="QTableWidget" name="URL_tableWidget">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>31</y>
        <width>1001</width>
        <height>381</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Creado</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fecha de Creacion</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="URL_pushButton_add">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>420</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="URL_pushButton_load">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>560</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cargar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="URL_pushButton_clear">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>560</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Borrar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="URL_pushButton_selectfolder">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>520</y>
        <width>231</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleccionar carpeta</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="URL_textEdit_path">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>479</y>
        <width>981</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
